src/Microsoft.XmlSerializer.Generator/src/dotnet-Microsoft.XmlSerializer.Generator.runtimeconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "runtimeOptions": {
+    "tfm": "netcoreapp2.0",
     "framework": {
       "name": "Microsoft.NETCore.App",
-      "version": "9.9.9"
```

Comment:
This is probably what you want for the actual product, but to test it you'll need one with 9.9.9.